var/spack/repos/builtin/packages/googletest/package.py

```diff
@@ -27,7 +27,8 @@ class Googletest(CMakePackage):
     version("1.7.0", sha256="f73a6546fdf9fce9ff93a5015e0333a8af3062a152a9ad6bcb772c96687016cc")
     version("1.6.0", sha256="5fbc058e5b662b9c86d93ac76fefb58eec89cbf26144b49669a38ecb62758447")
 
-    depends_on("cxx", type="build")  # generated
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
```

Comment:
@alalazo How were these determined? There's no C code in googletest...

Reply:
@sethrj See https://github.com/spack/spack/pull/47662#issuecomment-2484192915

Reply:
Hah, a common question then: guess CMake is sloppy about compiler checking. Thanks!

Reply:
See:

https://github.com/google/googletest/blob/d144031940543e15423a25ae5a8a74141044862f/CMakeLists.txt#L6C1-L6C33

From [CMake docs](https://cmake.org/cmake/help/latest/command/project.html):

> By default C and CXX are enabled if no language options are given. Specify language NONE, or use the LANGUAGES keyword and list no languages, to skip enabling any languages.

Reply:
Maybe we need a spackbot that automatically opens pull requests for sloppy packages 😂